drivers/w1/w1_zephyr_gpio.c

```diff
@@ -288,8 +288,7 @@ static int w1_gpio_init(const struct device *dev)
 	struct w1_gpio_data *data = dev->data;
 
 	if (gpio_is_ready_dt(spec)) {
-		int ret = gpio_pin_configure_dt(spec, GPIO_OUTPUT_INACTIVE | GPIO_OPEN_DRAIN |
-							      GPIO_PULL_UP);
+		int ret = gpio_pin_configure_dt(spec, GPIO_OUTPUT_INACTIVE | GPIO_OPEN_DRAIN | GPIO_OUTPUT | GPIO_INPUT );
 		if (ret < 0) {
```

Comment:
`GPIO_OUTPUT_INACTIVE` already includes the `GPIO_OUTPUT` flags so adding it again is redundant.
Also please keep below 100 chars per line and fix the spacing in that line.